### DISCUSSION TITLE
Zephyrus Duo 16 temp control

### DISCUSSION THREAD

**Discussion by h3XEnII** [seerge/g-helper#340] [Q&A]
Hello! First of all i'd like to say that the program is great. really great, thanks a lot for this. I'm using it on zephyrus duo 16 with 7945hx and 4090 laptop. I'm a bit struggling with temperatures control, let me try to explain. Disabling CPU boost, of course lower temperatures by a lot but i'm using that setting only when idle/casual usage. Apart from this, i've noticed keeping boost on either in "Enabled, Aggr, Efficient Enabled or Efficient Aggressive" will rise temperatures around 93-96C while gaming (game tested Mass Effect Legendary), which btw should be normal for that cpu. First question i'd would like to ask: is there is a way to lower temps without disabling cpu boost while gaming, (perhaps a sweet spot in cpu power limit slider) or i'm good with it? i'd like also to ask you some advices to keep under control temperatures and have good performance in general and lower temperatures, perhaps some different combinations with boost and wattage, because while playing i.e. FIFA (poorly CPU optimized game) it pushes the temp to thermal throttling (100C) even when not playing and even limiting power works only disabling boost. Tried different settings but seems to be too hot or too laggy due to power limiting/boost disable. Last, (but related to first question) from what i've understood, platform wattage is CPU+GPU right? In which combination should i set CPU and Platform slider if i'm in need? I hope i've explained myself, and that the thread is pertinent, i'm a bit new in this field (and a lot noob :D) and i would like to thank you in advance.

**Comment by tvud** [comment]
1. Undervolting both cpu and gpus, repasting and modifying the hard disk power management (leads to a considerable temperature reduction) may resolve your problem. 
2. Yes it is, platform wattage in your case is cpu+gpu.
3. Disregard it, i think it doesnt bring much benefit by adjusting it, it may be beneficial if some games/applications require significant resources from cpu or gpu.
